i18n/locales/en.ts

```diff
@@ -631,6 +631,7 @@ const translations = {
       nav_title: `Calculate the public key`,
       paragraph_one: `To derive a public key from a private key, we perform an elliptic curve operation repeatedly with the generator point. The generator point is a specific point on the curve. Its value is part of the secp256k1 standard and it's always the same:`,
       paragraph_two: `Mathematical operations on an elliptic curve are similar to addition. Therefore, repetition of those operations is similar to multiplication. We use the * symbol to describe the algorithm, where 'k' is the private key and 'P is the corresponding public key: `,
+      paragraph_three: `Some type-conversion hints`,
```

Comment:
```suggestion
      paragraph_three: `Type-conversion hints`,
```

After seeing it live I think we can shorten the heading of this accordion